File: 0229-majority-element-ii/0229-majority-element-ii.cs
public class Solution {
    public IList<int> MajorityElement(int[] nums) {
        Dictionary<int, int> dict = new Dictionary<int, int>();
        foreach (int i in nums) {
            if (dict.ContainsKey(i)) {
                dict[i]++;
            } else {
                dict[i] = 1;
            }
        }

        List<int> res = new List<int>();
        int compare = nums.Length / 3;
        foreach (var pair in dict) {
            if (pair.Value > compare) {
                res.Add(pair.Key);
            }
        }

        return res;
    }
    }

File: 0229-majority-element-ii/0229-majority-element-ii.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var majorityElement = function(nums) {
    const dict = {};
        for (const i of nums) {
            if (dict[i]) {
                dict[i]++;
            } else {
                dict[i] = 1;
            }
        }

        const res = [];
        const compare = nums.length / 3;
        for (const i in dict) {
            if (dict[i] > compare) {
                res.push(Number(i));
            }
        }

        return res;
};